editor/inspector/components/sprite.js

```diff
@@ -55,15 +48,10 @@ exports.ready = function() {
                     .then(async (spriteFrameMeta) => {
                         if (!spriteFrameMeta || !spriteFrameMeta.userData) { return; }
 
-                        let autoAtlasUuid = '';
-                        if (!spriteFrameMeta.userData.atlasUuid) {
-                            autoAtlasUuid = await findAutoAtlasFolder(spriteFrameUuid);
-                        }
-
                         const spriteAtlasDom = findDomByType(parentElement, 'cc.SpriteAtlas');
                         if (!spriteAtlasDom) { return; }
 
-                        spriteAtlasDom.value = spriteFrameMeta.userData.atlasUuid || autoAtlasUuid;
+                        spriteAtlasDom.value = spriteFrameMeta.userData.atlasUuid;
```

Comment:
The autoAtlasUuid may affect the build result and potentially increase atlas references, so it needs to be reverted.